modules/core/src/main/java/org/apache/ignite/internal/processors/rest/GridRestProcessor.java

```diff
@@ -1039,6 +1043,30 @@ private void startProtocol(GridRestProtocol proto) throws IgniteCheckedException
         X.println(">>>   handlersSize: " + handlers.size());
     }
 
+    /**
+     * Logs REST request processing result.
+     *
+     * @param req Received REST request.
+     * @param fut Result of REST request processing.
+     */
+    private void logRestRequestResult(GridRestRequest req, IgniteInternalFuture<GridRestResponse> fut) {
+        if (log == null || !log.isDebugEnabled())
+            return;
+
+        GridRestResponse resp = null;
+
+        IgniteCheckedException reqProcErr = null;
+
+        try {
+            resp = fut.get();
```

Comment:
Can we use `fut.error() == null` here or `fut.isCancelled()` and not use try catch logic?

Reply:
Done.